src/clj/vanilla/db/core.clj

```diff
@@ -17,11 +17,17 @@
 (hugsql/def-db-fns "sql/queries.sql")
 
 
-
+;; This database is stored locally. It is considered our "working" or dev db
+;; This should not be pushed to the repo or AWS, it will be added to git ignore
 (def vanilla-db
   "SQLite database connection spec."
   {:dbtype "sqlite" :dbname "vanilla_db"})
```

Comment:
make a new (def dbtype "sqlite") and use it in (def vanilla-db ...) and the (initialize-database ...) call so we only need to change it in one place if/when we change DBMSs

Reply:
Good idea, I added this and made sure both use it.